main.py

```diff
@@ -47,6 +56,17 @@ def fact(a):
     """Compute the Factorial of the number passed as argument."""
     return math.factorial(a)
 
+def binary(a):
+    """Compute the binary equivalent of the number passed as argument."""
```

Comment:
You may want to mention `decimal number` instead of just `number`? 
Same for hex and octal.